src/overset/TiogaOptions.C

```diff
@@ -41,11 +38,6 @@ TiogaOptions::load(const YAML::Node& node)
   if (node["node_resolution_multiplier"]) {
     nodeResMult_ = node["node_resolution_multiplier"].as<double>();
   }
-
-  if (node["adjust_mandatory_fringe_resolutions"]) {
```

Comment:
Should we delete the functions associated with this option as well?

Reply:
Yes, that would be more proper. Let me get that sorted out.

Reply:
I removed the boolean flag adjustResolutionsForFringes_, the function that checks it, and the flag checks themselves.